gulp/config.js

```diff
@@ -25,6 +25,11 @@ export default {
     dest: 'build/images'
   },
 
+  govuk_images: {
+    src: 'node_modules/govuk_frontend_toolkit/images/accordian-arrow.png',
```

Comment:
you should be able to do `path.dirname(require.resolve("govuk_frontend_toolkit"))` to get a safer path to the `govuk_frontend_toolkit`